### PR TITLE
Pharokka DataManager

### PR DESCRIPTION
**Pull request by paulzierep**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Tests and serving works locally, but `planemo --biocontainers` does not find the correct container, although the linter shows its there:
```
Applying linter biocontainer_registered... CHECK
.. INFO: BioContainer best-practice container found [quay.io/biocontainers/pharokka:1.2.1--hdfd78af_0].
```

### CODE REVIEW

**Comment by paulzierep** [comment]
@bgruening could you maybe help me out, as of why it cannot find the container ? 
```
galaxy.tool_util.deps.containers INFO 2023-04-14 16:01:24,687 [pN:main.1,p:115402,tN:LocalRunner.work_thread-1] Checking with container resolver [ExplicitContainerResolver[]] found description [None]
galaxy.tool_util.deps.containers INFO 2023-04-14 16:01:24,687 [pN:main.1,p:115402,tN:LocalRunner.work_thread-1] Checking with container resolver [CachedMulledDockerContainerResolver[namespace=biocontainers]] found description [None]
galaxy.tool_util.deps.containers INFO 2023-04-14 16:01:24,687 [pN:main.1,p:115402,tN:LocalRunner.work_thread-1] Checking with container resolver [CachedMulledDockerContainerResolver[namespace=local]] found description [None]
galaxy.tool_util.deps.containers INFO 2023-04-14 16:01:24,687 [pN:main.1,p:115402,tN:LocalRunner.work_thread-1] Checking with container resolver [MulledDockerContainerResolver[namespace=biocontainers]] found description [None]
galaxy.tool_util.deps.containers INFO 2023-04-14 16:01:24,687 [pN:main.1,p:115402,tN:LocalRunner.work_thread-1] Checking with container resolver [BuildDockerContainerResolver[namespace=local]] found description [None]
```
although the linter finds it:
```
Applying linter biocontainer_registered... CHECK
.. INFO: BioContainer best-practice container found [quay.io/biocontainers/pharokka:1.2.1--hdfd78af_0].
```

**Comment by bgruening** [comment]
Are you sure the container is not used? Have you tested that `install_databases.py` is not part of the container?

**Comment by paulzierep** [comment]
> 

Yes, when I pull the image locally I can run the command

File: data_managers/data_manager_pharokka_database_fetcher/data_manager/pharokka_db_fetcher.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<tool id="pharokka_db_fetcher" name="phrokka DB fetcher" tool_type="manage_data" version="0.0.3">
```
Comment (mvdbeek) [inline data_managers/data_manager_pharokka_database_fetcher/data_manager/pharokka_db_fetcher.xml:2]: All new tools should add profile version, that'll also allow containers to work.

Comment (paulzierep) [reply data_managers/data_manager_pharokka_database_fetcher/data_manager/pharokka_db_fetcher.xml:2]: @mvdbeek thank you very much !! profile did do the trick !!

File: data_managers/data_manager_pharokka_database_fetcher/tool_data_table_conf.xml.sample [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<tables>
+    <!-- Location of pharokka DB file -->
+    <table name="pharokka_db" comment_char="#">
+        <columns>value, dbkey, name, path</columns>
```
Comment (bgruening) [inline data_managers/data_manager_pharokka_database_fetcher/tool_data_table_conf.xml.sample:4]: maybe its a good idea to include something like a version? In the case tool are at some point not compatible with some DB versions?

Comment (paulzierep) [reply data_managers/data_manager_pharokka_database_fetcher/tool_data_table_conf.xml.sample:4]: good idea

**Comment by paulzierep** [comment]
checks failed installing R ?

File: data_managers/data_manager_pharokka_database_fetcher/data_manager_conf.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<data_managers>
+
+    <data_manager tool_file="data_manager/pharokka_db_fetcher.xml" id="pharokka_db_fetcher" version="1.2">
+        <data_table name="pharokka_db">
+            <output>
+                <column name="value" />
+                <column name="dbkey" />
+                <column name="name" />
```
Comment (bgruening) [inline data_managers/data_manager_pharokka_database_fetcher/data_manager_conf.xml:9]: is the version column here missing?

**Pull request merged**